lib/ui/platform_dispatcher.dart

```diff
@@ -168,6 +168,23 @@ class PlatformDispatcher {
   // A map of opaque platform view identifiers to view configurations.
   final Map<Object, ViewConfiguration> _viewConfigurations = <Object, ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
+  ///
+  /// This is used to bootstrap the framework for applications where the
+  /// platform provides a view, such as applications designed for
+  /// single-display mobile devices.
+  ///
+  /// While the [FlutterView]'s properties may change, this reference is
+  /// guaranteed to never change.
+  ///
```

Comment:
Should we add some kind of a warning here that applications and packages should not rely on this property being set and instead acquire the view via other means (e.g. `View.of(context)` is appropriate for most use cases)? This property is basically just interesting for low-level app bootstrapping.

Reply:
(I guess you're sort of hinting at this with the "see also" section)

Reply:
Could you take another look? I updated this comment to push people towards `View.of` instead. The new comment is less clear as to when an implicit view is provided by the platform though...

Reply:
I took a stab below at how I would write the documentation for this property. Feel free to copy it as a whole or bits and pieces that you like or ignore it completely.

---

The implicit [FlutterView] provided by the platform, if any.

If the platform provides an implicit view, it can be used to bootstrap the framework. This is common for platforms designed for single-view applications (e.g. on mobile devices with a single display). 

Applications and libraries must not rely on this property being set as it may be null depending on the engine's configuration. Instead, consider using [View.of] to lookup the [FlutterView] the current [BuildContext] is drawing into.

While the properties on the referenced [FlutterView] may change, the reference itself is guaranteed to never change over the lifetime of the application: If this property is null at startup, it will remain so throughout the entire lifetime of the application. If it points to a specific [FlutterView], it will continue to point to the same view until the application is shut down (although the engine may replace or remove the underlying backing surface of the view at its discretion).

See also:

 * [PlatformDisptacher.views] for a list of all [FlutterView]s provided by the platform.

Reply:
That's perfect, will update!

Reply:
Updated! FYI, I added `[View.of]` to the "See also" section. It's a little repetitive, but I want to nudge folks towards `View.of` as much as possible :)

Reply:
Added just a couple more suggestions above but this is looking pretty great!